### ISSUE TITLE
Remember last position

### ISSUE THREAD

**Issue by avidseeker** [organicmaps/organicmaps#7661]
⚠ Have you searched for similar, already existing issues? Yes

**Is your feature request related to a problem? Please describe.**

* GPS is turned off
* Select a POI on maps
* Close the app
* Running the app again shows the world map

**Describe the ideal solution**

Remember the last view/position.

**Describe alternatives you have considered**

Google Maps has this behavior, which makes sense because if users searched for a location, then they're more likely to search for a location nearby.

**Comment by biodranik** [comment]
The last screen position should be saved and working. Are you sure that Organic Maps does not crash in your case? How exactly are you closing the app? Does it work if you move it to the background before closing?

**Comment by avidseeker** [comment]
You're right. It seems that it needs to be moved to background first to get a chance to remember its position. If I immediately close the app without making it go to background first, it doesn't remember position.

**Comment by biodranik** [comment]
How and why are you closing the app without making it go to the background?

**Comment by avidseeker** [comment]
How? by swiping it out from recent apps menu.

Why? e.g: quick checking a place location or name.

It's just an old habit of me to immediately close apps I'm done with.

**Comment by Misalf-git** [comment]
On Android, it's not uncommon for apps to be unable to save settings when "force-closing" by swiping them out of recents instead of "exiting" first (via back button).

**Comment by biodranik** [comment]
So you press a button that shows recent apps first, and then close OM, right? Then we need to check why the location/settings are not saved.

**Comment by avidseeker** [comment]
> On Android, it's not uncommon for apps to be unable to save settings when "force-closing" by swiping them out of recents instead of "exiting" first (via back button).

That's interesting, but I doubt that doing this mounts to "force-closing". I thought this is possible from "Force stop" within the app info. 

> a button 

Actually, a gesture (like the iPhone gestures), but I doubt it will make a difference.

**Comment by RedAuburn** [comment]
i feel like this is more a responsibility of the system fusedocationprovider
